diff --git a/configure.ac b/configure.ac
index d57c5d2..3ca64db 100644
--- a/configure.ac
+++ b/configure.ac
@@ -71,9 +71,6 @@ dnl ================================================================
 
 IT_PROG_INTLTOOL([0.40.0])
 
-AM_GNU_GETTEXT_VERSION([0.17])
-AM_GNU_GETTEXT([external])
-
 GETTEXT_PACKAGE=libpeas
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE,"$GETTEXT_PACKAGE", [Gettext package])
